templates/components/InlineEditInteraction.html.twig

```diff
@@ -1,87 +1,48 @@
 <div {{ attributes.defaults(stimulus_controller('inline-edit')) }}>
 
-    {% set finished=false %}
-    {% set modalId=unique_id('interaction-', false) %}
 
-    {% if interaction is not null %}
-        {% if interaction.finished %}
-            {% set finished=true %}
-        {% endif %}
+    <div class="btn-group-vertical btn-group-sm w-100 my-2">
+    {% for case in enum('App\\Enum\\ReadStatus').cases() %}
```

Comment:
I don't understand how this works.
It loops over all read statuses so it shows one button per status ?
missa no comprendo